src/views/reducers/index.ts

```diff
@@ -0,0 +1,8 @@
+const todos = (state: any = [], action: any): any => {
```

Comment:
For my information what are `todos` in Expresso's context

Reply:
Bad copy/path from a react sample.

Reply:
Ok